openmls/src/framing/plaintext.rs

```diff
@@ -335,6 +335,18 @@ impl MlsPlaintext {
     pub fn authenticated_data(&self) -> &[u8] {
         self.authenticated_data.as_slice()
     }
+
+    #[cfg(test)]
+    pub(crate) fn tamper_with_signature(&mut self) {
```

Comment:
maybe something like invalidate?

```suggestion
    pub(crate) fn invalidate_signature(&mut self) {
```

Reply:
If this exists here, `signature_mut` shouldn't be necessary.